common/spec/spec_helper.rb

```diff
@@ -85,6 +85,32 @@ def fixture(*name)
   File.read(File.join("spec", "fixtures", File.join(*name)))
 end
 
+# Creates a temporary directory and writes the provided files into it.
+#
+# @param files [DependencyFile] the files to be written into the temporary directory
+def write_tmp_repo(files,
```

Comment:
Since the NuGet updater needs files on disk for parsing and updating, this was super useful to convert existing tests where the test file contents were defined inline.